Kitodo-DataManagement/src/main/java/org/kitodo/data/database/persistence/apache/ProjectManager.java

```diff
@@ -14,11 +14,12 @@
 import java.sql.SQLException;
 import java.util.List;
 
-import org.apache.log4j.Logger;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 import org.kitodo.data.database.beans.ProjectFileGroup;
 
 public class ProjectManager {
-    private static final Logger logger = Logger.getLogger(MySQLHelper.class);
+    private static final Logger logger = LogManager.getLogger(MySQLHelper.class);
```

Comment:
Can you please correct the used class name from `MySQLHelper.class` to `ProjectManager.class` as the class name is `ProjectManager`?